docs/contributing/review-process.md

```diff
@@ -0,0 +1,172 @@
+  [reviewable pull requests](../contributing/reviewable-prs.md) guide for
+  advice.
+
+- If you have a question that you expect to be resolved during the review
+  process, put it in a PR comment attached to a relevant part of the changes.
+  The review process will go a lot more smoothly if points of uncertainty
+  are explicitly laid out.
+
+- Make sure that the pull request passes all CI tests. You can sometimes
+  request initial feedback if there are open questions that will impact how
+  you update the tests. But in general, maintainers will wait for your PR to
+  pass tests before reviewing your work.
+
+If any part of your contribution is from someone else (code
+snippets, images, sounds, or any other copyrightable work, modified or
+unmodified), be sure to review the instructions on how to [properly
+attribute][licensing] the work.
+
+If your PR was not ready for review when
+you first posted it (e.g., because it was failing tests, or you
+weren't done working through the self-review checklist), notify maintainers when
+you'd like them to take a look by posting a quick "Ready for review!" comment on
+the main GitHub thread for your PR.
+
+[git-guide-make-pr]: ../git/pull-requests.md
+[licensing]: ../contributing/licensing.md
+
+### Draft pull requests
+
+If it helps your workflow, you can submit your pull request marked as
+a [draft][github-help-draft-pr] while you're still working on it. When ready for
+review:
+
+1. Make sure your PR is no longer marked as a [draft][github-help-draft-pr], and
+   doesn't have "WIP" in the title.
+
+1. Post a quick "Ready for review!" comment on the main GitHub thread for your
+   PR.
+
+[github-help-draft-pr]: https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-pull-requests#draft-pull-requests
+
+## Stages of a pull request review
+
+This section describes the stages of the pull request review process. Each stage
+may require several rounds of iteration. Don't feel daunted! Not every PR will
+require all the stages described, and the process often goes quite quickly for
+smaller changes that are clearly explained.
+
+1. **Product review.** Oftentimes, seeing an initial implementation will make it
+   clear that the product design for a feature needs to be revised, or that
+   additional changes are needed. The reviewer may therefore ask you to amend or
+   change the implementation. Some changes may be blockers for getting the PR
+   merged, while others may be improvements that can happen afterwards. Feel
+   free to ask if it's unclear which type of feedback you're getting.
+   (Follow-ups can be a great next issue to work on!)
+
+   Your PR might be assigned the [product
+   review](https://github.com/zulip/zulip/pulls?q=is%3Aopen+is%3Apr+label%3A%22product+review%22)
+   label in this phase, or later in the review process as questions come up. You
```

Comment:
"this phase" ... is that referring to phase 1 as the bullet point?

Reply:
Yeah. Changing to "at this stage" to make the terminology consistent.